### PR TITLE
Update getting started guide to include directions for ouroboros

### PR DESCRIPTION
**Pull request by swenson**
and running `make`

### CODE REVIEW

File: docs/intro/getting-started.rst [hunk 1]
```diff
@@ -16,6 +16,12 @@ The first step is to create a project directory, and clone Batavia:
     $ cd tutorial
     $ git clone https://github.com/pybee/batavia.git
 
+Batavia now requires a copy of Ouroboros (the Python standard library, written in Python) to build, so we also need to clone that.
```
Comment (freakboy3742) [inline docs/intro/getting-started.rst]: No need for the tense here - someone coming to the project for the first time won't have the context of "before".

Comment (freakboy3742) [inline docs/intro/getting-started.rst]: A similar instruction should also be needed for the developer instructions; that section creates a completely new checkout and environment, so a fresh ouroboros checkout will also be required.

Comment (swenson) [inline docs/intro/getting-started.rst]: Commit added to address these comments.

**Comment by swenson** [comment]
Thanks!

**Pull request merged**